Composer/packages/client/src/recoilModel/utils/skill.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
delete commented

Reply:
please remove commented code in this file.

Reply:
it's auto generated by lint. Whenever I save, it will be added.